docs/hub/repositories-getting-started.md

```diff
@@ -0,0 +1,224 @@
+
+To learn about Git branching, you can try out the [Learn Git Branching interactive tutorial](https://learngitbranching.js.org/).
+
+### Using tags
+
+Git allows you to *tag* commits so that you can easily note milestones in your project. As such, you can use tags to mark commits in your Hub repos! To learn about using tags, you can visit [this DevConnected post](https://devconnected.com/how-to-create-git-tags/).
+
+Beyond making it easy to identify important commits in your repo's history, using Git tags also allows you to [clone a repository at a specific tag](https://www.techiedelight.com/clone-specific-tag-with-git/). The `huggingface_hub` library also supports working with tags, such as [downloading files from a specific tagged commit](https://huggingface.co/docs/huggingface_hub/main/en/how-to-downstream#hfhuburl).
+
+### How to duplicate or fork a repo (including LFS pointers)
+
+If you'd like to copy a repository, depending on whether you want to preserve the Git history there are two options.
+
+#### Duplicating without Git history
+
+In many scenarious, if you want your own copy of a particular codebase you might not be concerned about the previous Git history. In this case, you can quickly duplicate a repo with the handy [Repo Duplicator](https://huggingface.co/spaces/osanseviero/repo_duplicator)! You'll have to create a User Access Token, which you can read more about in the [security documentation](TODO).
+
+#### Duplicating with the Git history (Fork)
+
+A duplicate of a repository with the commit history preserved is called a *fork*. You may choose to fork one of your own repos, but it also common to fork other people's projects if you would like to tinker with them.
+
+When you want to fork or [rebase](https://git-scm.com/docs/git-rebase) a repository with [LFS](https://git-lfs.github.com/) files you cannot use the usual Git approach that you might be familiar with since you need to be careful to not break the LFS pointers. Forking can take time depending on your bandwidth because you will have to fetch and re-upload all the LFS files in your fork.
+
+For example, say you have an upstream repository, **upstream**, and you just created your own repository on the Hub which is **myfork** in this example.
+
+1. Create a destination repository (e.g. **myfork**) in https://huggingface.co 
+
+2. Clone your fork repository
+
+```
+git lfs clone https://huggingface.co/me/myfork.git
+```
+
+3. Fetch non LFS files
+
+```
+cd myfork
+git lfs install --skip-smudge --local # affects only this clone
+git remote add upstream https://huggingface.co/friend/upstream.git
+git fetch upstream
+```
+
+4. Fetch large files. This can take some time depending on your download bandwidth
+
+```
+git lfs fetch --all upstream # this can take time depending on your download bandwidth
+```
+
+4.a. If you want to completely override the fork history (which should only have an initial commit), run:
```

Comment:
Since we have a colon here, I think it'd be nice to also add a colon for all the other steps to maintain consistency.